dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -14,7 +14,10 @@ class ApplicationsControllerTest < ::ActionController::TestCase
 
       @workshop_admin = create :workshop_admin
       @workshop_organizer = create :workshop_organizer
-      @regional_partner = create :regional_partner, program_managers: [@workshop_organizer]
+      @regional_partner = create :regional_partner,
+        program_managers: [@workshop_organizer],
+        cohort_capacity_csd: 25,
+        cohort_capacity_csp: 25
```

Comment:
Make these different numbers so we can differentiate in the results, make sure they're not being mixed up.